index.html

```diff
@@ -1,78 +1,160 @@
+            <h6>Bring in a project or use inhouse recording so we can help enhance its quality so you get the best sound possible</h6>
+
+          </div>
+          <div class="rehearsal-space">
+            <img src="Images/Rehearsal-space.jpg" alt="rehearsal-space">
+            <h3>Rehearsal Space.</h3>
+            <h6>Bring in your band before recording or before a show for a private rehearsal space with top quality instruments.</h6>
+
+          </div>
+          <div>
+            <img src="Images/musicians.png" alt="muscian">
+            <h3>Session Musicians.</h3>
+            <h6>You can get hold of our professional session musicians incase you need a band for your session</h6>
+
+          </div>
+          <div>
+            <img src="Images/song-writer.png" alt="writer">
+            <h3>Song Writers. </h3>
+            <h6>Your song lacks lyrics worry not we have our own song writers that can help you with lyrics</h6>
+          </div>
+          <div>
+            <img src="Images/vocalist.png" alt="backing-mics">
+            <h3>Backing Vocals</h3>
+            <h6>Looking to harmonize your music with the best vocalis? Worry not, We have vocalist that sing all different ranges</h6>
+          </div>
+        </div>
+        <div class="submit-b">
+          <button type="submit">Book A Session</button>
         </div>
       </div>
     </section>
 
 <!-- about section -->
 
-<section>
+<section id="musicians">
   <div class="cardz-sector">
     <div>
       <h2>Meet Our Resident Musicians.</h2>
     </div>
 
-    <div class="cards">
-      <!-- <img src="Images/bass-guitarist.jpg" alt="musician">
-      <div class="text">
-        <h3>Kenya Malaika</h3>
-        <hr>
-        <p>Is a professional bassist, who majors in jazz and classical music.</p>
-        <h6>She has experience also as a composer,vocal arranger,song writer and music director</h6> -->
-      </div>
+    <div class="card-list">
+
+      <div class="dynamic">
+   
+      </div>  
       
-    </div>  
+    </div>
+
+    <div class="seemore">
+      <button>See less <img src="Images/arrow_down 1.png" alt="arrow-up"></button>
+    </div>
+ 
 </section>
 
-<section>
-  <div class="About-us"></div>
-</section>  
+   <footer>
+    <div class="partners">
+      <div class="parthead">
+        <h3>Partners</h3>
+        
+      </div>
+
+      <div class="sponsors">
+        <ul>
+          <li><img src="Images/mozilla.png" alt="mozilla"></li>
+          <li><img src="Images/googlekorea.png" alt="googlekorea"></li>
+          <li><img src="Images/naver.png" alt="naver"></li>
+          <li><img src="Images/daumkakaologo.png" alt="daumkakaologo"></li>
+          <li><img src="Images/airbnb.png" alt="airbnb"></li>
+        </ul>
+      </div>
+    </div>
+    <div class="license">
+      <img src="Images/Logo.png" alt="logo">
+      <h2>
+        Groovy studios.
+        All Rights reserved
+      </h2>
+    </div>
+  
+</footer>
```

Comment:
<table>
- [ ] Kindly include the `footer` section of the design on your homepage as demonstrated on the provided design as this will make your design complete and create a good user experience.

  <tr>
    <td align="middle">Your design</td>
     <td align="middle">Provided design</td>
  </tr>
  <tr>
    <td valign="top"><img width="372" alt="Screenshot 2023-04-24 at 21 39 45" src="https://user-images.githubusercontent.com/107192076/234124269-1b9b93f4-8cc8-41be-a2a0-b83171f4c368.png" width="30%"></td>
<td align="top">
<img width="374" alt="Screenshot 2023-04-24 at 21 43 26" src="https://user-images.githubusercontent.com/107192076/234126495-9d7fa6e2-c45b-4870-8e17-2329750cb9ec.png">
</td>
  </tr>
- [ ] Please include the `partner` section to make your design complete as demonstrated in the provided design.
<tr>
    <td align="middle">Your design</td>
     <td align="middle">Provided design</td>
  </tr>
<tr>
<td align="top">
<img width="369" alt="Screenshot 2023-04-24 at 22 09 41" src="https://user-images.githubusercontent.com/107192076/234128039-a9d09158-09d8-444f-8e0d-aaada3038c58.png">
</td>
<td align="top">
<img width="369" alt="Screenshot 2023-04-24 at 22 12 07" src="https://user-images.githubusercontent.com/107192076/234128075-d01a687d-1807-4393-9cf2-602c3b6800b1.png">

</td>
</tr>
 </table>

Reply:
Hello, Thank you for the review, I really appreciate it. I noticed you suggested  I add a footer on the homepage, and the design suggests otherwise and I am not sure if you actually noticed the design. There Is no footer on the design, I wanted you to confirm That and help me with a way forward. The about me does have a footer with a partners section. Please revisit the design and tell me where to go from there.

Reply:
@God-Logic sorry for the late reply. The home section of the design doesn’t show the `footer` section because the design was showing a specific section of the page but the footer section should be there. Please try to think outside of the box and replicate the behavior of a real-world web application. For example, if you visit a site like [BBC](https://www.bbc.com/) you can see the footer is a static section across all pages.